### ISSUE TITLE
[CLOSED] Implement support for training with mixed val objectives

### ISSUE THREAD

**Issue by jeswan** [nyu-mll/jiant-v1-legacy#311]
<a href="https://github.com/W4ngatang"><img src="https://avatars0.githubusercontent.com/u/5520155?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [W4ngatang](https://github.com/W4ngatang)**
_Tuesday Jul 31, 2018 at 20:46 GMT_
_Originally opened as https://github.com/nyu-mll/jiant/pull/311_

----

Allow for mixed validation objectives during pretraining. Our strategy is when we are training on more than one task, for a metric `m` that is supposed to decrease, we use `1 - (m / dec_val_scale)` instead of `m`, where `dec_val_scale` is a hyperparameter I'm currently tuning. So if we're training on LM and SST, the macro average is `.5 * (SST_acc + (1 - ppl / dec_val_scale)`. By default, `dec_val_scale = 100`, which I interpret as 10 points in perplexity is roughly weighted as much as 10% absolute accuracy.

----

_**[W4ngatang](https://github.com/W4ngatang)** included the following code: https://github.com/nyu-mll/jiant/pull/311/commits_
